Core/ServiceLocator.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Autofac;
+
+namespace CKAN
+{
+    /// <summary>
+    /// This class exists as a really obvious place for our service locator (ie: Autofac container)
+    /// to live.
+    /// </summary>
+    public static class ServiceLocator
+    {
+        private static IContainer _container;
+        public static IContainer container
+        {
+            get
```

Comment:
It's probably worth noting in a comment that this isn't thread-safe, not that it needs to be, but if CKAN ever goes super-parallel it's something to watch out for.